include/bm/bm_sim/P4Objects.h

```diff
@@ -323,13 +330,27 @@ class P4Objects {
                                        const std::string &src_conditional_name,
                                        const std::string &edge_name,
                                        const std::string &dst_node_name);
+  void change_register_array_size_rt(const std::string& name,
+                                     const std::string& register_array_size);
+  void change_register_array_bitwidth_rt(const std::string& name,
+                                         const std::string& register_array_bitwidth);
   void flex_trigger_rt(bool on);
   void delete_flex_rt(const std::string &pipeline_name,
                       const std::string &name);
   void delete_conditional_rt(const std::string &pipeline_name,
                              const std::string &name);
   void delete_match_table_rt(const std::string &pipeline_name,
                              const std::string &name);
+  void delete_register_array_rt(const std::string& name);
+  void rehash_register_array(const std::string& target_register_array,
```

Comment:
This is a function with many parameters. Need to add comments talking about what each parameter does.

Reply:
first_hash_function, second_hash_function, third_hash_function, is a bit wired. Does this mean this function require only 3 hash functions or it could need more?